import/index_java/src/org/vufind/index/DateTools.java

```diff
@@ -26,19 +26,45 @@
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
  * Date indexing routines.
  */
 public class DateTools
 {
+    private final static Pattern YEAR_PATTERN = Pattern.compile("-?\\d{1,4}");
+    private final static Pattern BC_YEAR_PATTERN = Pattern.compile("([0-9]+) [Bb][.]?[Cc][.]?");
```

Comment:
Maybe worth accounting for the possibility of space between B. and C. as well?

```suggestion
    private final static Pattern BC_YEAR_PATTERN = Pattern.compile("([0-9]+) [Bb][.]?\s*[Cc][.]?");
```

Reply:
I think this one comment may have gotten missed.